package.json

```diff
@@ -81,15 +73,8 @@
 		"typescript": "^4.3.5",
 		"yarn": "^1.22.10"
 	},
-	"files": [
```

Comment:
It seems that the fiels field is missing.

Reply:
Either .npmignore or package.json files should be used. https://stackoverflow.com/questions/37857149/is-the-files-property-necessary-in-package-json